packages/plans-grid-next/src/components/header-price.tsx

```diff
@@ -172,20 +170,19 @@ const PlanFeatures2023GridHeaderPrice = ( {
 		return (
 			<HeaderPriceContainer>
 				{ ! current && (
-					<Badge className="plan-features-2023-grid__badge" isForIntroOffer>
+					<Badge className="plan-features-2023-grid__badge">
 						{ translate( 'Limited Time Offer' ) }
 					</Badge>
 				) }
 				{ isLargeCurrency ? (
```

Comment:
With "only" the changes below, you will only show the cross-out pricing when `isLargeCurrency` is `true`. I think what you want here is to remove this condition and have it work identical to the `isGridPlanOneTimeDiscounted` below (since intro offers will now always show cross-out price)?

Reply:
I think it is same case as commented previously: https://github.com/Automattic/wp-calypso/pull/86434#discussion_r1467455621

Reply:
> I think what you want here is to remove this condition and have it work identical to the isGridPlanOneTimeDiscounted below (since intro offers will now always show cross-out price)?

I did a bit of testing here and this does work well and will enforce that both intro offers and discount prices use the same pricing format (i.e. small, crossed out price pairs).

We could also refactor `isGridPlanOnIntroOffer` and `isGridPlanOneTimeDiscounted` into one render block but maybe its cleaner just to have them separate alongside `isAnyVisibleGridPlanOneTimeDiscounted || isAnyVisibleGridPlanOnIntroOffer`.

Reply:
@jeyip I think fixing the large-currency inconsistency is the only part left to address (and a rebase obviously)